web/components/AccessManagement/RoleSelector.tsx

```diff
@@ -53,7 +53,7 @@ function RoleSelector(props: PropertyType) {
       onOpen={onOpen}
       onClose={onClose}
       placement="bottom"
-      offset={[42, 0]}
+      offset={[0, 0]}
```

Comment:
Attribute is the same for both breakpoints, so just assign `offset=0`

Reply:
It doesn't let you have a one parameter offset, but because the default is 0 anyway, I just removed that attribute.